sdks/python/apache_beam/runners/portability/flink_runner_test.py

```diff
@@ -400,6 +400,9 @@ def test_callbacks_with_exception(self):
   def test_register_finalizations(self):
     raise unittest.SkipTest("BEAM-6868")
 
+  def test_custom_merging_window(self):
```

Comment:
we may need to add this to spark_runner_test.py as well.

Reply:
Done.